README.md

```diff
@@ -5,7 +5,14 @@ as an example plugin package and can be used as starting point to create your ow
 
 Please visit http://gionkunz.github.io/chartist-js/plugins.html for more information.
 
+## Download 
+The easiest way to get started, using Bower
+```
+bower install chartist-plugin-pointlabels --save
+```
+
 ## Available options and their defaults
+bower install chartist-plugin-pointlabes --save
```

Comment:
Hey there. Thanks for the PR. Why would you include the bower install instruction also in the available options section? Cheers